staging/src/k8s.io/apiserver/pkg/endpoints/apiserver_test.go

```diff
@@ -3424,6 +3424,12 @@ func TestCreate(t *testing.T) {
 		t.Errorf("unexpected error: %v %#v", err, response)
 	}
 
+	la := make(map[string]string)
```

Comment:
Why did you have to update these tests?

Reply:
They were failing because without my change it seemed like the lastApplied field was nil

Reply:
And that way it gets tested that the field is as it should be, or am I wrong about that?

Reply:
No, that's good, let's keep it this way